Core/src/main/java/com/paypal/android/core/API.kt

```diff
@@ -41,4 +58,19 @@ class API internal constructor(
         }
         return clientID
     }
+
+    suspend fun sendAnalyticsEvent(name: String) {
```

Comment:
can this be `private` it will not be used by merchants

Reply:
I tried both `private` and `internal` but it broke compilation since it's currently used by `CardAPI`.